lib/ffmprb/filter.rb

```diff
@@ -242,6 +242,7 @@ def volume_exp(volume)
         prev_vol = volume[prev_at] || 1.0
         exp = "#{volume[volume.keys.last]}"
         volume.each do |at, vol|
+          next if at == prev_at
```

Comment:
I see it for cases where ducking audio with silence at the beginning.
eq: `should duck the overlay sound wrt the main sound` test

`rspec spec/ffmprb_spec.rb:619` for current state of master will produce it as part of the command used for audio ducking e.q.:
`if(between(t, 0.0, 0.0), 0.1, 0.9)` 

full volume expression is:
`if(between(t, 11.6612, 12.6612), (0.8*t + -9.22896)/1.0, if(between(t, 8.44567, 11.6612), 0.1, if(between(t, 7.44567, 8.44567), (-0.8*t + 6.856536)/1.0, if(between(t, 4.641590000000001, 7.44567), 0.9, if(between(t, 3.6415900000000003, 4.641590000000001), (0.8*t + -2.8132720000000004)/1.0000000000000004, if(between(t, 0.0, 3.6415900000000003), 0.1, if(between(t, 0.0, 0.0), 0.1, 0.9)))))))`